workflows/linters.yml

```diff
@@ -0,0 +1,30 @@
+name: Linters
```

Comment:
Hi @adamilare 👍,

Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!

so-close

![well-1](https://user-images.githubusercontent.com/103386183/196423863-0f522555-1b03-455e-a452-b822cdcfc398.gif)

Highlights
- Set up with api only backend added correctly ✔️
- Linters ✔️

Required Changes ♻️
 - I did not understand the purpose of the ```workflows/linters.yml``` in the codebase.  Did you for get to remove after adding one from the github folder?